specs-go/config.go

```diff
@@ -2,9 +2,6 @@ package specs
 
 import "os"
 
-// CurrentVersion is the current version of the Spec.
```

Comment:
I think we should keep the version defined here.

Reply:
I'm OK to move it to `version.go` too. Thanks for confirming the behaviour in the OCI runtime spec.